data-serving/data-service/src/index.ts

```diff
@@ -1,3 +1,11 @@
+// Set up appmetrics-dash before importing additional dependencies.
+// This ensures that the module captures metrics for dependent systems, like
+// MongoDB.
+import Dash from 'appmetrics-dash';
+if (process.env.NODE_ENV !== 'test') {
+    Dash.attach();
```

Comment:
can we move the import in that if as well to avoid importing it if we don't need to?

Reply:
The Typescript compiler is smart enough to do this, IIUC:

https://www.typescriptlang.org/docs/handbook/modules.html#optional-module-loading-and-other-advanced-loading-scenarios